jbpm-designer-client/src/test/java/org/jbpm/designer/client/shared/OnResizeTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2015 JBoss Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.designer.client.shared;
+
+import com.google.gwt.user.client.ui.Widget;
+import com.google.gwtmockito.GwtMock;
+import com.google.gwtmockito.GwtMockitoTestRunner;
+import org.jbpm.designer.client.DesignerPopUpPresenter;
+import org.jbpm.designer.client.DesignerViewImpl;
+import org.jbpm.designer.client.DesignerWidgetPresenter;
+import org.jbpm.designer.client.DesignerWidgetView;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+import org.mockito.Spy;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+@RunWith(GwtMockitoTestRunner.class)
+public class OnResizeTest {
+
+    @Spy
+    private DesignerWidgetView designerWidgetView;
+
+    @Spy
+    private DesignerPopUpPresenter designerPopupPresenter;
+
+    @Spy
+    private DesignerViewImpl designerViewImpl;
+
```

Comment:
This (DesignerViewImpl) is not being used. I guess you wanted to write a test for it too?